### ISSUE TITLE
Data transforms in finetuning

### ISSUE THREAD

**Issue by junkunyuan** [YanzuoLu/MALE#4]
Hi, 
I see the imagenet default mean=[0.485, 0.456, 0.406] and std=[0.229, 0.224, 0.225] are used in finetuning in MALE.
https://github.com/YanzuoLu/MALE/blob/2ab99b3e0f01a8e23ab985a9365e8c9c002094ab/utils/transforms.py#L35

But TransReID and TransReID-SSL use mean=[0.5, 0.5, 0.5] and std=[0.5, 0.5, 0.5].
https://github.com/damo-cv/TransReID/blob/f1e1f8f1e3efb526f4b91698c037a47955ee3496/configs/Market/vit_transreid.yml#L23
https://github.com/damo-cv/TransReID-SSL/blob/fc39e88240aa7cb7b28dd2097e7f161ae2be3ad8/transreid_pytorch/configs/market/vit_base_baseline.yml#L19

May I know why you modified this?

Thank you very much!

**Comment by YanzuoLu** [comment]
In fact, we are not following TransReID or TransReID-SSL methodologically, but we have noticed this difference. If I remember correctly, I should have tried these two in the fine-tuning stage on the MAE pre-training model and finally chose ImageNet's as the default for better performance. I don't know why TransReID and TransReID-SSL do this, I think you can also perform experiments and try it yourself if you are interested.

**Comment by YanzuoLu** [comment]
Maybe you can open an issue under TransReID or TransReID-SSL repo? I'm also curious about this question.

**Comment by junkunyuan** [comment]
Thanks! I am on it.

**Comment by junkunyuan** [comment]
BTW, if you did not follow TransReID or TransReID-SSL, then which work did you follow to perform the downstream ReID task?

Because I found the downstream ReID training configuration of MALE is quite different from TransReID-SSL and other works I know.

**Comment by YanzuoLu** [comment]
Our follow work can be considered as the baseline proposed by TransReID. But actually the design of ID Loss + Triplet Loss is very common, and it is not a novel thing. For details, please refer to [fast-reid](https://github.com/JDAI-CV/fast-reid).
